backend/app/GraphQL/Validators/SubmissionUpdateInputValidator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+namespace App\GraphQL\Validators;
+
+use Nuwave\Lighthouse\Validation\Validator;
+
+class SubmissionUpdateInputValidator extends Validator
```

Comment:
Rename this file and class to UpdateSubmissionInputValidator and it'll follow the convention established by lighthouse.

Reply:
Refactored as suggested